src/foam/u2/PropertyBorder.js

```diff
@@ -184,9 +184,9 @@
            */
           addClass(this.myClass('errorText')).
           enableClass(this.myClass('colorText'), colorSlot).
-          // show(errorSlot.and(modeSlot.map(m => m == foam.u2.DisplayMode.RW))).
+           show(errorSlot.and(modeSlot.map(m => m == foam.u2.DisplayMode.RW))).
```

Comment:
Why this change?

Reply:
because there is too much space on fund your balance view (below the choices and above from) because of the error text space.